lib/honeycomb/context.rb

```diff
@@ -13,6 +13,15 @@ def current_span
       spans.last
     end
 
+    def current_span_context(span: current_span)
+      trace_id = span.trace.id
+      span_id = span.id
+      trace_fields = span.trace.fields
+      dataset = span.dataset
+
+      [trace_id, span_id, trace_fields, dataset]
```

Comment:
Really not a fan of returning a four element array here as opposed to a hash or a class instance. What happens if we want to add/remove/change this? Also, this is the propagation context, isn't it? Why is this called `current_span_context` instead of `propagation_context`?

Reply:
I know :( @paulosman I'm curious your thoughts here. I opted to keep this consistent with existing patterns but I don't love this being an array rather than a hash with named params. Worth changing?